xl/player/player.py

```diff
@@ -397,7 +397,8 @@ def is_playing(self):
             :returns: whether the player is currently playing
             :rtype: bool
         """
-        return self._engine.get_state() == 'playing'
+        return self._engine.get_state() is not None \
```

Comment:
> Oh, that means it's `self._engine` that is None at that point.

Yes.

I'm not quite sure whether it is allowed that `self._engine` is `None`.  This might also be a bug in PreviewDevice plugin. What do you think?